react/example-customer-account--loyalty-app--react/extensions/customer-account-loyalty-extension/src/CustomerFulfillmentDetailsDelivery.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+Card,
+BlockStack,
+reactExtension,
+Button,
+useApi,
+TextBlock
+} from "@shopify/ui-extensions-react/customer-account";
+
+export default reactExtension(
+"customer-account.order-status.fulfillment-details.render-after",
+( ) => <CustomerFulfillmentDetailsDelivery />
+);
+
+function CustomerFulfillmentDetailsDelivery( ) {
+  const {i18n} = useApi<"customer-account.order-status.fulfillment-details.render-after">()
+  return (
+    <Card padding>
```

Comment:
I don't think we should use a card here, then it is card within card, for fulfullment card, we use horizontal line to divide each section  

<img width="607" alt="Screenshot 2023-10-31 at 11 27 36" src="https://github.com/Shopify/customer-account-tutorials/assets/12724940/4afc0c10-3218-411f-8892-f9020e7507dd">